src/main/people.ts

```diff
@@ -0,0 +1,99 @@
+import { getBasecampUrl, sendPaginatedBasecampGetRequest } from "./basecamp";
+import { PeopleNotPopulatedError } from "./error/peopleNotPopulatedError";
+import { PersonMissingIdError } from "./error/personMissingIdError";
+import { PersonNameIdMapNotCachedError } from "./error/personNameIdMapNotCachedError";
+import { getScriptProperty, setScriptProperty } from "./propertiesService";
+
+type PersonNameIdMap = {[key: string]: string};
+
+// Project ID hardcoded to our SD Basecamp Integration project for testing
+// In the future we may consider moving this to a script property as a config value
+const PROJECT_ID: string = "38736474";
+const PEOPLE_PATH: string = `/projects/${PROJECT_ID}/people.json`;
+const PEOPLE_MAP_KEY: string = "PEOPLE_MAP";
+
+// Regex to match a person's name followed by a city. ex. John Doe (SD)
+const CITY_REGEX: string = "^(.+?) (.+?) \((.+)\)$";
```

Comment:
i suspect this won't work for filipino ppl like "Jonathan Koby Cayaban (SD)". i think it's safe to just chop off the parentheses portion and everything before that is the name.

Reply:
Updated the matching to match any string that comes before `()` so we can just chop off the parentheses